.github/workflows/merge-queue.yml

```
@@ -1,12 +1,173 @@
+    needs:
+      - prepare
+    if: needs.prepare.outputs.E2E_CHUNKS
+    runs-on: arc-runners
+    timeout-minutes: 45
+    env:
+      AFFECTED_PROJECT: ${{ matrix.projects }}
+      CYPRESS_PROJECT_ID: 4q7jz8
+      CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
+      API_MOCKS: 'true'
+      NODE_OPTIONS: --max-old-space-size=4096
+      E2E_BUILD_ID: '${{ needs.prepare.outputs.E2E_BUILD_ID }}-${{ github.run_attempt }}'
```

actionlint (1.7.4)

property "e2e_build_id" is not defined in object type {base_sha: string; cache_keys: string; e2e_chunks: string; head_sha: string; test_chunks: string}

(expression)